crates/uv-resolver/src/lock/mod.rs

```diff
@@ -1263,7 +1263,11 @@ impl Lock {
         }
 
         doc.insert("package", Item::ArrayOfTables(packages));
-        Ok(doc.to_string())
+
+        let content = doc.to_string();
+        Ok(format!(
+            "# This file was @generated by uv.  Do not edit by hand.\n{content}"
```

Comment:
Could we use a single space after the period? Basically every single style guide on earth recommends a single space (even when using a monospace font).
e.g. see https://cmosshoptalk.com/2020/03/24/one-space-or-two/ or https://apastyle.apa.org/style-grammar-guidelines/paper-format/accessibility/typography#myth4